### PR TITLE
01. Responsive website

### PR DESCRIPTION
**Pull request by franpossetto**

### CODE REVIEW

File: prepros.config [hunk 1]
```diff
@@ -0,0 +1,932 @@
+{
```
Comment (franpossetto) [inline prepros.config]: What is it?

File: src/pages/Home.tsx [hunk 2]
```diff
@@ -9,41 +9,35 @@ export const Home = () => {
...
-    }
-  }
-
   return (
     <>
       <main id="mainHome" className="container">
-        <section className="container d-flex flex-column justify-content-between">
-          <div className="row d-flex justify-content-between mainHomeRow1">
-            <div className="col-sm-6">
+        <section className="container d-flex flex-column justify-content-between mainHomeContainer">
```
Comment (franpossetto) [inline src/pages/Home.tsx]: convert all css custom class from camelCase to kebab-case. On this way we follow the same convention than bootstrap. add the prefix "nina" to identify nina custom classes. For example:

`nina-main-home-container`.

don't use numbers to differentiate classes with the same name.  For example:

`mainHomeRow1`

Delete classes that are duplicated.

Comment (manuelserafin) [reply src/pages/Home.tsx]: done

File: prepros.config (hunk 1 above)
Comment (manuelserafin) [reply prepros.config]: Sorry, it is for using SCSS but it is not necessary because SCSS is not used.

**Pull request merged**